github/resource_github_repository_webhook.go

```diff
@@ -115,13 +109,13 @@ func resourceGithubRepositoryWebhookCreate(d *schema.ResourceData, meta interfac
 	// GitHub returns the secret as a string of 8 astrisks "********"
 	// We would prefer to store the real secret in state, so we'll
 	// write the configuration secret in state from our request to GitHub
-	if hook.Config["secret"] != nil {
-		hook.Config["secret"] = hk.Config["secret"]
+	if hook.Config.Secret != nil {
+		hook.Config.Secret = hk.Config.Secret
 	}
 
-	hook.Config = insecureSslStringToBool(hook.Config)
+	// hook.Config = insecureSslStringToBool(hook.Config)
```

Comment:
This comment should be removed entirely rather than left in the PR; same for the similar line on 176.